src/applications/disability-benefits/2346/app-entry.jsx

```diff
@@ -0,0 +1,13 @@
+import '../../../platform/polyfills';
```

Comment:
Do we need a copy of App-entry for this? Or can we leverage the one thats already in the application?

Reply:
> Do we need a copy of App-entry for this? Or can we leverage the one thats already in the application?

Not sure, this was generated by the form generator.